server/server.js

```diff
@@ -37,8 +37,46 @@ const ACTIVE_SHOPIFY_SHOPS = {};
 
 const WEBHOOK_TRIGGER_TIMESTAMPS = {};
 
+const LINKED_FLOWS = {};
+
 const shopIdFromShop = (shop) => shop.split(".")[0];
 
+async function getAllLinkedFlows() {
+  // In order to get all linked flows, we're making this request using "default" as "user_id"
+  const url = new URL(
+    `${process.env["HG_API_URL"]}/${process.env["HG_ENV_ID"]}/flows/linked`
```

Comment:
@davi-souza We should be using the supported flows endpoint for this, that's the whole point of it